config/producer.properties

```diff
@@ -23,7 +23,7 @@ bootstrap.servers=localhost:9092
 # specify the compression codec for all data generated: none, gzip, snappy, lz4, zstd
 compression.type=none
 
-# name of the partitioner class for partitioning events; default partition spreads data randomly
+# name of the partitioner class for partitioning records; The default uses "sticky" partitioning logic, which attempts to fill batches sent to each broker in order to improve throughput.
```

Comment:
Small improvement? I thought we should say something about balance.

> The default uses "sticky" partitioning logic which spreads the load evenly between partitions, but improves throughput by attempting to fill the batches sent to each partition.

Reply:
a ha, you're right. I only focus on the comparison between sticky and round-robin partitioner. Updated.